### PR TITLE
doc: contin_array: add missing docs

### PR DESCRIPTION
**Pull request by greg-fer**
Added library doc page for the continuous array library. OCT-2332.

Signed-off-by: Grzegorz Ferenc <Grzegorz.Ferenc@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/libraries/others/contin_array.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+.. contents::
+   :local:
+   :depth: 2
+
+.. TODO: Add description
+
+Configuration
+*************
+
+To enable the library, set the :kconfig:option:`CONFIG_CONTIN_ARRAY` Kconfig option to ``y`` in the project configuration file :file:`prj.conf`.
```
Comment (koffes) [inline doc/nrf/libraries/others/contin_array.rst:27]: For now, there is no such setting. The functionality is simply available, but I may be corrected on that in the original PR.

File: include/contin_array.h [hunk 2]
```diff
@@ -9,20 +9,30 @@
 
 #include <zephyr/kernel.h>
 
-/** @brief. Creates a continuous array from a finite array
- * @param pcm_cont		Pointer to the destination array
+/**
```
Comment (koffes) [inline include/contin_array.h:12]: Suggestion: This function takes an array which the user wants to loop over (could e.g. be a period of a Pulse-code modulated (PCM) sine wave). This is supplied to pcm_finite and pcm_finite_size. The user may then call this function which populates pcm_cont with pcm_cont_size bytes. This lets users create N repetitions of pcm_finite into pcm_cont.
The function keeps track of the current position in pcm_finite so the function can be called multiple times and maintain the correct position in pcm_finite. Wrapping is handled automatically.

File: doc/nrf/libraries/others/contin_array.rst (hunk 1 above)
Comment (greg-fer) [reply doc/nrf/libraries/others/contin_array.rst:27]: So how can you enable the library, if it is missing its Kconfig?

**Comment by greg-fer** [comment]
@koffes , added description to the page. Please answer my question about how to enable the library, and I think we can then merge this PR into yours.

File: doc/nrf/libraries/others/contin_array.rst (hunk 1 above)
Comment (koffes) [reply doc/nrf/libraries/others/contin_array.rst:27]: It is always available. - Added by default

**Review by koffes — APPROVED** [review]
I can make the final changes as I get comments on the PR.

**Pull request merged**